Mathlib/Computability/MyhillNerode.lean

```diff
@@ -0,0 +1,105 @@
+/-
+Copyright (c) 2024 Google. All rights reserved.
```

Comment:
Why Google?

Reply:
It's company policy. There's a process I can follow that will let me use my own name there, but it's complicated and not really worth fighting for. Note that this is still in my personal time (not work).

Reply:
Ah, cool! I just wanted to make sure it was intentional that Google is written there.